lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -296,22 +299,22 @@ def sync(self):
     def modify(self):
         """Modify the cache cluster. Note it's only possible to modify a few select options."""
         nodes_to_remove = self._get_nodes_to_remove()
-        try:
-            response = self.conn.modify_cache_cluster(cache_cluster_id=self.name,
-                                                  num_cache_nodes=self.num_nodes,
-                                                  cache_node_ids_to_remove=nodes_to_remove,
-                                                  cache_security_group_names=self.cache_security_groups,
-                                                  cache_parameter_group_name=self.cache_parameter_group,
-                                                  security_group_ids=self.security_group_ids,
-                                                  apply_immediately=True,
-                                                  engine_version=self.cache_engine_version)
-        except boto.exception.BotoServerError as e:
-            self.module.fail_json(msg=e.message)
-
-        self._refresh_data()
+        if not self.module.check_mode:
+            try:
+                response = self.conn.modify_cache_cluster(cache_cluster_id=self.name,
+                                                      num_cache_nodes=self.num_nodes,
```

Comment:
This wasn't aligned correctly before but we may as well fix it up now we're changing it.